internal/internal_workflow_client_test.go

```diff
@@ -1176,6 +1176,35 @@ func (s *workflowClientTestSuite) TestStartWorkflow_WithDataConverter() {
 	s.Equal(createResponse.GetRunId(), resp.RunID)
 }
 
+func (s *workflowClientTestSuite) TestStartWorkflow_ByteBypass() {
+	// default DataConverter checks for []byte args, and does not re-encode them.
+	// this is NOT a general feature of DataConverter, though perhaps it should be
```

Comment:
previously I thought this _was_ a general thing, but no!  apparently not.

this might be something we can change in v2, since I don't _think_ anyone really depends on it... or at least change the default.  an explicit "no, encode byte slices too" seems fine, it's just unfortunate to lose the escape hatch without opting into it.